### PR TITLE
Small resort of 70-mouse.hwdb

### PR DESCRIPTION
**Pull request by claell**
One step for #10940

### CODE REVIEW

**Comment by poettering** [comment]
CI doesn't like this.

/cc @whot

File: hwdb/70-mouse.hwdb [hunk 1]
```diff
@@ -292,18 +292,18 @@ mouse:usb:v047dp1020:*Kensington Expert Mouse*
 # Lenovo
 ##########################################
 
-# Lenovo Optical USB Mouse
+# Lenovo USB mouse model MO28UOL
+mouse:usb:v04b3p310c:name:USB Optical Mouse:
+ MOUSE_DPI=400@142
+ 
+ # Lenovo Optical USB Mouse
```
Comment (whot) [inline hwdb/70-mouse.hwdb]: no preceding space here please,  that's probably why the CI fails

**Comment by poettering** [comment]
also needs a rebase

File: hwdb/70-mouse.hwdb (hunk 1 above)
Comment (claell) [reply hwdb/70-mouse.hwdb]: Thanks! Fixed.

**Comment by claell** [comment]
I will take care of the rebase thing probably tomorrow.

**Comment by claell** [comment]
Since I just edited the file in GitHub and did not know how to easily rebase there I created a new PR for this: #10995

**Pull request closed without merging**